City-of-Helsinki/drupal-employment-services
# THF-501: THF-501: remove extra words

Removed words below from synonyms plugging.


#### Testing

- check that the words are no longer in the code
- run `drush search-api:clear; drush search-api:rebuild-tracker; drush search-api:index; drush cr;`
- check that the search still working

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/maria.ahokas/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/maria.ahokas/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

--
"ammatti barometristä,   ammattibarometristä",
"asia   kirjasta, asiakirjasta",
"asiointi ohjeet,   asiointiohjeet",
"duuni tori, duunitori",
"hakemus lomake, hakemuslomake",
koulutus ala, koulutusala
"koulutus palvelu,   koulutuspalvelu",
"lupakortti   koulutus, lupakortti koulutus, lupakorttikoulutus",
"matka kulu, matkakulu",
"muutoksen hakuohje, muutoksenhaku   ohje, muutoksenhakuohje",
"neuvonta palvelu,   neuvontapalvelu",
"nollatunti sopimus,   nollatuntisopimus",
"oleskelu   lupa, oleskelulupa",
"opiskelu paikka,   opiskelupaikka",
"oppimis vaikeus,   oppimisvaikeus",
"oppisopimus paikka,   oppisopimuspaikka",
"osaamis kartoitus,   osaamiskartoitus",
"soitto pyyntö, soittopyytö",
"työnhaku sivusto, työn haku sivusto,   työnhakusivusto",
"työpaikka ilmoitus, työ paikka   ilmoitus, työpaikkailmoitus",
"työpaikka, työ paikka",
"kassan raha, ansioon suhteutettu   päiväraha",
"korttikoulutus,  lupakorttikoulutus",
"työ,   työnhaku",
"työpaikka,   työnhaku",
"työkkäri,  kuntakokeilu",
"työkkäri,  oma asiointi",
"työkkäri,  neuvonta",
"työkkäri,  verkkopalvelu",
"työkkäri,  puhelinpalvelut",
"työttömyysraha,  ansioon suhteutettu päiväraha",
"töihin,  työnhaku",
"töitä, työnhaku",
"te-palvelut,   neuvonta",
"te-palvelut,   verkkopalvelu",
"te-palvelut,   puhelinpalvelut",
"te-toimisto,   oma asiointi",
"te-toimisto,   neuvonta",
"te-toimisto,   verkkopalvelu",
"te-toimisto,   puhelinpalvelut",



</body>

</html>
